precompile/params.go

```diff
@@ -9,6 +9,11 @@ import "github.com/ethereum/go-ethereum/common"
 const (
 	ModifyAllowListGasCost = 20_000
 	ReadAllowListGasCost   = 5_000
+
+	MintGasCost = 20_000
+	BurnGasCost = 20_000
+
+	CallNewAccountGas = 25000 // Paid for CALL when the destination address didn't exist prior.
```

Comment:
This seems excessive given that a send to any address in ETH (even if it creates a new account is 21k).

Reply:
this is same as https://github.com/ava-labs/subnet-evm/blob/master/params/protocol_params.go#L41

Reply:
removed this